components/Card.js

```diff
@@ -21,13 +21,29 @@ export default function Card({
         </DeleteButton>
       )}
       <p>Date: {date}</p>
-      {weight && <p>Weight: {weight}</p>}
-      {height && <p>Height: {height}</p>}
-      {feastTime && (
-        <p>
-          Nurse time: {feastTime.substr(0, 2)}:{feastTime.substr(2, 2)}:
-          {feastTime.substr(4, 2)}
-        </p>
+      {weight?.[0] && (
+        <ul>
+          {weight.map((weight, index) => (
+            <li key={index}>Weight: {weight} </li>
+          ))}
+        </ul>
+      )}
+      {height?.[0] && (
+        <ul>
+          {height.map((height, index) => (
+            <li key={index}>Height: {height} </li>
+          ))}
+        </ul>
+      )}
+      {feastTime?.[0] && (
+        <ul>
+          {feastTime.map((entry, index) => (
```

Comment:
feast means something like eating at a big buffet or restaurant, I guess you mean feedingTime here?

Reply:
I guess this was just a poor joke from my side :(